cmd/virtctl/BUILD.bazel

```diff
@@ -46,3 +46,29 @@ go_binary(
     visibility = ["//visibility:public"],
     x_defs = version_x_defs(),
 )
+
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_image",
+)
+
+container_image(
+    name = "version-container",
+    base = "//:passwd-image",
+    directory = "/",
+    files = ["//:get-version"],
+)
+
+container_image(
```

Comment:
I added this so we also publish virtctl in a container image.
This I used to do in-cluster load tests and I think generally doesn't hurt as a way to distribute the CLI.
Could be moved to a separate PR if needed.

Reply:
i'm not opposed to exporting a load test image. i'm not sure the load test belongs in virtctl though.